### PR TITLE
add text and list, table, form

### PR DESCRIPTION
**Pull request by MashaLoban**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -1,27 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
Comment (ghost) [inline about/index.html]: Bad idea.

File: about/index.html [hunk 2]
```diff
@@ -1,27 +1,38 @@
...
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Об авторе/title>
 </head>
+
```
Comment (ghost) [inline about/index.html]: See the previous comment.

File: about/index.html [hunk 3]
```diff
@@ -1,27 +1,38 @@
...
+    <h2>Потому, что мне кажется это <em>интересным.</em></h2>
+    </p>
+    <br />
+    <p>
+    <h3>Потому, что мне кажется это <mark>перспективным.</mark></h3>
+    </p>
+    <br />
+    <p>
+    <h4>Потому, что в случае, если мне будет интересно программирование, и это будет приносить <ins>доход</ins>, то я
+        буду немножко счастливее, чем сейчас.<br /> Т.к. буду знать, что у меня получилось добиться результата. <del>Но
```
Comment (ghost) [inline about/index.html]: We have discussed `br` at the previous lesson.

File: about/index.html [hunk 4]
```diff
@@ -1,27 +1,38 @@
...
-    <p><h2>Потому, что мне кажется это <em>интересным.</em></h2></p>
-    <br/>
-    <p><h3>Потому, что мне кажется это <mark>перспективным.</mark></h3></p>
-    <br/>
-    <p><h4>Потому, что в случае, если мне  будет интересно программирование, и это будет приносить <ins>доход</ins>, то я буду немножко счастливее, чем сейчас.<br/> Т.к. буду знать, что у меня получилось добиться результата. <del>Но это не точно.</del> И немножко добавим (H<sub>2</sub>O)<sup>2</sup>.</h4></p>
-    <br/>
+    <hr />
+    <br />
+    <p>
+    <h2>Потому, что мне кажется это <em>интересным.</em></h2>
```
Comment (BubentsovKirill) [inline about/index.html:20]: Don't put `headers` into other inline tags.
2021-04-04_00h43_06

File: about/index.html [hunk 5]
```diff
@@ -1,27 +1,38 @@
...
+    </p>
+    <br />
+    <p>
+    <h3>Потому, что мне кажется это <mark>перспективным.</mark></h3>
```
Comment (ghost) [inline about/index.html:19]: See the previous comment.

File: about/index.html [hunk 6]
```diff
@@ -1,27 +1,38 @@
...
+    </p>
+    <br />
+    <p>
```
Comment (ghost) [inline about/index.html]: Again

File: index.html [hunk 7]
```diff
@@ -1,13 +1,156 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
Comment (ghost) [inline index.html]: Delete an empty line.

File: index.html [hunk 8]
```diff
@@ -1,13 +1,156 @@
...
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ghost) [inline index.html]: Close tags.

File: index.html [hunk 9]
```diff
@@ -1,13 +1,156 @@
...
     <title>Моя первая страница</title>
+    <style>
+        table,
+        th,
+        td {
+            border: 1px solid;
+        }
+    </style>
 </head>
+
```
Comment (ghost) [inline index.html]: Delete empty line.

File: index.html [hunk 10]
```diff
@@ -1,13 +1,156 @@
...
+        semper,
+        netus a praesent condimentum donec adipiscing magna. Bibendum fames adipiscing in
+        arcu parturient dignissim, eu varius rutrum enim feugiat sem, dictum donec potenti mattis iaculis.</p>
+    <p>Nullam &#9200 &#9994 sociosqu morbi facilisis nulla etiam posuere suspendisse, odio finibus vehicula cursus
+        sollicitudin
+        dignissim malesuada, bibendum mollis taciti erat enim torquent. Ultrices primis mi
+        orci torquent potenti inceptos phasellus netus habitant etiam, erat ante sed finibus blandit quisque elit
+        adipiscing senectus viverra, parturient mollis aliquam himenaeos nascetur in venenatis
+        nisl sodales.</p>
+    <img src="D:\docs\Маша\Курсы intellectfox\Домашка\1.jpg" alt="цветы" />
```
Comment (ghost) [inline index.html]: Bad practice to gets some files outside your project directory. You have 2 ways, add the file to your directory or provide some url.

File: about/index.html (hunk 1 above)
Comment (MashaLoban) [reply about/index.html]: hi, i don't touch about/index.html file yesterday . this mistakes maybe automatically appeared when I pressed "format on save". I  didn't do it manually.

File: about/index.html (hunk 2 above)
Comment (MashaLoban) [reply about/index.html]: done

File: about/index.html (hunk 3 above)
Comment (MashaLoban) [reply about/index.html]: fixed

File: index.html (hunk 8 above)
Comment (MashaLoban) [reply index.html]: done

File: index.html (hunk 10 above)
Comment (MashaLoban) [reply index.html]: fixed

File: index.html (hunk 9 above)
Comment (MashaLoban) [reply index.html]: ok

File: index.html (hunk 7 above)
Comment (MashaLoban) [reply index.html]: it's not me too, it's automatically. fixed

File: about/index.html (hunk 6 above)
Comment (MashaLoban) [reply about/index.html]: mayby fixed

File: about/index.html (hunk 5 above)
Comment (MashaLoban) [reply about/index.html:19]: done

**Pull request merged**